tests/e2e/specs/miscellaneous/CustomOpenVSXRegistry.spec.ts

```diff
@@ -0,0 +1,221 @@
+		});
+
+		test('Wait the workspace readiness', async function (): Promise<void> {
+			await projectAndFileTests.waitWorkspaceReadinessForCheCodeEditor();
+		});
+
+		test('Check if a project folder has been created', async function (): Promise<void> {
+			Logger.debug(`new SideBarView().getContent().getSection: get ${testRepoProjectName}`);
+			projectSection = await projectAndFileTests.getProjectViewSession();
+			expect(await projectAndFileTests.getProjectTreeItem(projectSection, testRepoProjectName), 'Project folder was not imported').not
+				.undefined;
+		});
+		test('Accept the project as a trusted one', async function (): Promise<void> {
+			await projectAndFileTests.performTrustAuthorDialog();
+		});
+		test('Remove VSX plugins and set redhat.vscode-xml', function (): void {
+			// modify the openvsx-sync.json file with just one item - for speeding up the build an image
+			kubernetesCommandLineToolsExecutor = e2eContainer.get(CLASSES.KubernetesCommandLineToolsExecutor);
+			kubernetesCommandLineToolsExecutor.workspaceName = workspaceName;
+			kubernetesCommandLineToolsExecutor.loginToOcp('admin');
+			kubernetesCommandLineToolsExecutor.getPodAndContainerNames();
+			const pathToOpenVSXOriginFile: string = `${pathToPluginRegistry}openvsx-sync.json`;
+			const pathToOpenVSXTmpFile: string = `${pathToPluginRegistry}temp.json`;
+			const editVSCListCommand: string = `jq "[.[] | select(.id == \\"redhat.vscode-xml\\")]" ${pathToOpenVSXOriginFile} > ${pathToOpenVSXTmpFile} && mv ${pathToOpenVSXTmpFile} ${pathToOpenVSXOriginFile}`;
+			const output: ShellString = kubernetesCommandLineToolsExecutor.execInContainerCommand(editVSCListCommand);
+			expect(output.code).to.equals(0);
+			const getCheClusterNamespaceCommand: string = 'oc get checluster --all-namespaces -o json | jq -r ".items[0].metadata.name"';
+			const getCheClusterNameSpace: string = 'oc get checluster --all-namespaces -o json | jq -r ".items[0].metadata.namespace"';
+
+			// get che cluster name and namespace and init variables
+			cheClusterName = kubernetesCommandLineToolsExecutor.execInContainerCommand(getCheClusterNamespaceCommand).replace('\n', '');
+			cheClusterNamespace = kubernetesCommandLineToolsExecutor.execInContainerCommand(getCheClusterNameSpace).replace('\n', '');
+		});
+
+		test('Login podman into official registry', function (): void {
+			const commandToAuthenticateInRegistry: string = `podman login -u \"${process.env.REGISTRY_LOGIN}\" -p ${process.env.REGISTRY_PASSWORD} registry.redhat.io`;
+			const output: ShellString = kubernetesCommandLineToolsExecutor.execInContainerCommand(commandToAuthenticateInRegistry);
+			expect(output).contains('Login Succeeded!');
+		});
+
+		test('Build and push custom image', function (): void {
+			currentNamespace = kubernetesCommandLineToolsExecutor.namespace;
+			const podmanPushCommand: string = `podman push ${internalRegistry}/${currentNamespace}/che-plugin-registry:${BASE_TEST_CONSTANTS.TESTING_APPLICATION_VERSION}`;
```

Comment:
Let's set `next` version by default in main branch: https://github.com/eclipse-che/che/blob/main/tests/e2e/constants/BASE_TEST_CONSTANTS.ts#L74

Here is an example of removal of redundant digit of bug-fix version: https://github.com/eclipse-che/che/blob/main/tests/e2e/specs/dashboard-samples/Documentation.spec.ts#L43

Reply:
thanks for the correction